pkg/plugin/plugin.go

```diff
@@ -240,3 +240,17 @@ func (p *plugin) GetControlPlanePods(ctx context.Context, oc *api.OpenShiftManag
 	}
 	return json.Marshal(pods)
 }
+
+func (p *plugin) UpdateWithPluginConfig(ctx context.Context, cs *api.OpenShiftManagedCluster, deployFn api.DeployFn, pluginTemplate *pluginapi.Config) *api.PluginError {
```

Comment:
do not pass in `pluginTemplate *pluginapi.Config` here, access it via `p.config`

Reply:
bump